datamegh/api/storage/s3/s3.py

```diff
@@ -0,0 +1,56 @@
+"""Class to access AWS S3 service extends base class AWS"""
```

Comment:
Leading and training spaces required for all docstrings - for cleanliness.
```suggestion
""" Class to access AWS S3 service extends base class AWS """
```